app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -43,65 +44,59 @@ const Notes = () => {
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
+    <>
+      <MenuBar showMenu={showMenu} />
+      <Container>
+        <Header
+          title="All Notes"
+          actionBlock={
+            <Button
+              className="neeto-ui-bg-gray-800 rounded-sm hover:bg-gray-600"
+              icon="ri-add-line"
+              label="Add new note"
+              size="small"
+              onClick={() => setShowNewNotePane(true)}
+            />
+          }
+          menuBarToggle={() => {
+            setShowMenu(showMenu => !showMenu);
+          }}
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+            placeholder: "Search Name, Email, Phone Number, Etc.",
+          }}
+        />
+        {notes.length ? (
+          <>
```

Comment:
Do we need this fragment over here? What happens when we remove it? Can you create a video and show me the issue from UI?

Reply:
@yedhink This was a mistake from my side, as I missed this part of the code in self-reviewing. 

Here is the link to the video
https://www.loom.com/share/521ae5fa97ca48d3b0ed23154fdfb518

Reply:
Got it. That's fine.

Reply:
By the way, didn't Eslint raise an errors regarding this piece of code? As per https://github.com/bigbinary/wheel/blob/f4d3f66b49754e2b547a5f924a7d57f1449e5235/.eslint-rules/react.js#L34 I believe an error should have been raised.

But lemme know so that we can look further into the rule.

Reply:
![Desktop 2022-09-21 13-58-20](https://user-images.githubusercontent.com/44156359/191455186-9774deb0-5cb4-4553-9c88-b1730b2e1b2e.png)

Eslint is not raising any error regarding this piece of code, even if there is a useless fragment.